run_dir/design/project_summary.html

```diff
@@ -0,0 +1,76 @@
+{% extends "base.html" %}
+
+<!--
+Template file: project_summary.html
+URL: /project_summary/<project>
+Title: displays the project summary(ies) of a project
+Description: Shows a table with all project summaries.
+-->
+
+{% block stuff %}
+
+<h1 id="{{ project_id }}" class="project_page_heading"><span id="page_title">Project Summary</span> <small>project summary for <a href="/project/{{ project_id }}">{{ project_id }}</a></small></h1>
+
+<div id="page_content" >
+<div id="summary-list">
+    {% for oneproc in processes%}
+    <h2>Project summary {{ oneproc.id }}</h2>
+    <input type="button" value="save process udf" class="save_pc_udf btn btn-primary btn-md pull-right" id="save_{{ oneproc.id }}" data-toggle="modal" data-target="#updateModal"/>
+    <table class="table table-striped table-bordered" id="summary_udfs_{{ oneproc.id}}">
+    <tr>
+    {% for udf in step_fields %}
+        <th class="step_{{oneproc.id}}_name">{{ udf }}</th>
+    {% end %}
+    </tr>
+    <tr>
+    {% for udf in step_fields %}
+        {% if isinstance(oneproc.udf.get(udf, '-'), basestring) and len(oneproc.udf.get(udf, '-')) <= 10 %}
+            <td><input class='step_{{oneproc.id}}_value_{{ udf.replace(" ", "_") }} form-control' type='text' value="{{ oneproc.udf.get(udf, '-') }}" /></td>
+        {% else %}
+            <td><textarea class='step_{{oneproc.id}}_value_{{ udf.replace(" ", "_") }} form-control' cols="12" > {{ oneproc.udf.get(udf, '-') }} </textarea></td>
```

Comment:
Need to remove the spacing around the variable name, as the HTML has spaces on either side of the string otherwise. eg:
```html
<textarea ...>{{ oneproc.udf.get(udf, '-') }}</textarea>
```
not:
```html
<textarea ...> {{ oneproc.udf.get(udf, '-') }} </textarea>
```
Same goes for other `textarea` fields.